README.md

```diff
@@ -1,65 +1,50 @@
+# Obsidian AutoMOC
+#### Based on [Zoottelkeeper](https://github.com/akosbalasko/zoottelkeeper-obsidian-plugin) 0.6.0
 
+**This currently works seperate from Zoottelkeeper but I'm hoping to integrate it with the main plugin.**
 
-# ZoottelKeeper: A Zookeeper for your Zettelkasten folder

```

Comment:
Don't you like the name? :D Your's is definitely more self-explainable (?) but I think we cannot change the name of the package and the plugin just like that, because we loose the continuity with the previous versions in Obsidian. May I ask you to switch it back? Of course, with keeping you among the authors, as I tested you did an amazing job! :)

Reply:
Yes! I'll switch it back! To be honest, I changed it because I wasn't sure you'd accept the PR. Since you like the changes, I think we should really just stick with Zoottelkeeper and I'll let everything there the way it was 👍🏼